pulp_container/app/tasks/builder.py

```diff
@@ -113,7 +113,7 @@ def build_image_from_containerfile(
     containerfile = Artifact.objects.get(pk=containerfile_pk)
     repository = ContainerRepository.objects.get(pk=repository_pk)
     name = str(uuid4())
-    with tempfile.TemporaryDirectory(".") as working_directory:
```

Comment:
First kwarg is not `dir`, it's `suffix`, even though the former would make way more sense : /